src/app/formFields/getFormFieldsValidationSchema.ts

```diff
@@ -20,11 +19,15 @@ const ERROR_KEYS: { [fieldName: string]: string } = {
     phone: 'address.phone_number',
 };
 
-export default memoize(function getAddressValidationSchema({
+export interface FormFieldValues {
+    [key: string]: string | { [id: string]: any };
+}
+
+export default memoize(function getFormFieldsValidationSchema({
```

Comment:
I think it is the same situation here. The translation keys are still tied to address fields (see `ERROR_KEYS`). We need to find a way to pull them out so they are not statically coupled with this function.

Reply:
similar to what I described above, some of the form fields are shared like `first_name` and `last_name`, so don't think it's worth moving the translations key a level above. I think for now it's okay to leave the mapping in this class and if it gets more complex in the future we could always refactor it.